### ISSUE TITLE
[Bug]: 'bool' object is not subscriptable - Unlisted playlists / Channels with no uploads

### ISSUE THREAD

**Issue by gremi-jr** [tubearchivist/tubearchivist#614]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read the [how to open an issue](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#how-to-open-an-issue) guide, particularly the [bug report](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#bug-report) section.

### Operating System

Linux (Debian 12 Bookworm)

## Describe the bug
I've tried to add [this playlist](https://www.youtube.com/playlist?list=PLNqhmeogCubHUjReodrMmocjjZfainClf) which includes unlisted videos. But I get the error "'bool' object is not subscriptable". The Playlist then shows up but with no Thumbnail and cant be accessed.

<img width="1120" alt="image" src="https://github.com/tubearchivist/tubearchivist/assets/50887574/d1eac50d-6077-40c1-8797-1290c8a80e9a">
<img width="1120" alt="image" src="https://github.com/tubearchivist/tubearchivist/assets/50887574/78f0f4c8-c882-4eff-88ca-e27028e56428">

The same happens when I create a unlisted playlist in my own account which has no uploads.
It seems like you can not add an playlist from a channel which has no video uploads?

## Steps To Reproduce
Use the given playlist above or create an unlisted playlist in a youtube channel with no uploads and add videos from other channels.

## Expected behavior
You should be able to subscribe playlists even when the youtube channel who has created the playlist has no uploads. 

My use-case: I have a "archive" playlist where I put all videos in that I want to archive. My channel has no video uploads but I want to save my playlist anyway.   

### Relevant log output

```shell
[2023-11-24 18:28:51,282: INFO/MainProcess] Task subscribe_to[a8d89945-e40a-4f04-a5ba-d4a79aa4a092] received
[2023-11-24 18:28:51,283: WARNING/ForkPoolWorker-16] subscribe_to create callback
[2023-11-24 18:28:51,309: WARNING/ForkPoolWorker-16] PLNqhmeogCubHUjReodrMmocjjZfainClf: get metadata from es
[2023-11-24 18:28:51,527: WARNING/ForkPoolWorker-16] UCuUHDELv0TZM7VeDchmZrFQ: get metadata from es
[2023-11-24 18:28:51,531: WARNING/ForkPoolWorker-16] {"_index":"ta_channel","_id":"UCuUHDELv0TZM7VeDchmZrFQ","found":false}
[2023-11-24 18:28:51,531: WARNING/ForkPoolWorker-16] UCuUHDELv0TZM7VeDchmZrFQ: get metadata from youtube
[2023-11-24 18:28:51,980: WARNING/ForkPoolWorker-16] ERROR: [youtube:tab] UCuUHDELv0TZM7VeDchmZrFQ: This channel has no uploads
[2023-11-24 18:28:51,980: WARNING/ForkPoolWorker-16] https://www.youtube.com/channel/UCuUHDELv0TZM7VeDchmZrFQ: failed to get info from youtube with message ERROR: [youtube:tab] UCuUHDELv0TZM7VeDchmZrFQ: This channel has no uploads
[2023-11-24 18:28:51,982: WARNING/ForkPoolWorker-16] a8d89945-e40a-4f04-a5ba-d4a79aa4a092 Failed callback
[2023-11-24 18:28:51,984: ERROR/ForkPoolWorker-16] Task subscribe_to[a8d89945-e40a-4f04-a5ba-d4a79aa4a092] raised unexpected: TypeError("'bool' object is not subscriptable")
Traceback (most recent call last):
  File "/root/.local/lib/python3.11/site-packages/celery/app/trace.py", line 477, in trace_task
    R = retval = fun(*args, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^
  File "/root/.local/lib/python3.11/site-packages/celery/app/trace.py", line 760, in __protected_call__
    return self.run(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/home/tasks.py", line 356, in subscribe_to
    SubscriptionHandler(url_str, task=self).subscribe(expected_type)
  File "/app/home/src/download/subscriptions.py", line 347, in subscribe
    self.subscribe_type(item, expected_type=expected_type)
  File "/app/home/src/download/subscriptions.py", line 357, in subscribe_type
    PlaylistSubscription().process_url_str([item])
  File "/app/home/src/download/subscriptions.py", line 198, in process_url_str
    self.channel_validate(playlist_h.json_data["playlist_channel_id"])
  File "/app/home/src/download/subscriptions.py", line 216, in channel_validate
    channel.build_json(upload=True)
  File "/app/home/src/index/channel.py", line 43, in build_json
    self.process_youtube_meta()
  File "/app/home/src/index/channel.py", line 51, in process_youtube_meta
    self.youtube_meta["thumbnails"].reverse()
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
TypeError: 'bool' object is not subscriptable
[2023-11-24 18:28:51,985: WARNING/ForkPoolWorker-16] a8d89945-e40a-4f04-a5ba-d4a79aa4a092 return callback
```

**Comment by ItsNoted** [comment]
I am getting this when just trying to add new channels. Never seen this before.

**Comment by trafficman** [comment]
This is the same behavior and error I was getting in #587 (I can confirm that you also cannot add playlists from channels with no public uploads to your download queue)

Perhaps the implemented workaround doesn't cover channels with no videos for whatever reason?

**Comment by edv** [comment]
I run into the same problem with my personal playlists (my YouTube channel does not have any uploaded videos).

What happens behind the scenes is that Tube Archivist retrieves information on the channel the playlist belongs to using yt-dlp. Because the channel does not contain any video's, yt-dlp returns an error `This channel has no uploads` and stops (causing Tube Archivist to fail, as it relies on channel information, even for playlists).

Not sure what the best approach would be to resolve this problem, but hopefully this info gives a bit more insight on where the problem is caused in the application.

**Comment by bbilly1** [comment]
OK, workaround take 2: parsing the `/featured` page looks like is our best way forward, as even a channel without videos will still have a featured page that yt-dlp can parse as before. Unstable build is out, please help with verifying.

**Comment by JasonLandbridge** [comment]
@bbilly1 I'm on unstable now and I tried adding this channel: https://www.youtube.com/@Fireship to the download queue but received the following error:

```bash
[2023-12-03 11:15:34,944: INFO/MainProcess] Task extract_download[56b049b9-ce3b-4f99-a189-3c6343458a9f] received
[2023-12-03 11:15:34,946: WARNING/ForkPoolWorker-16] extract_download create callback
[2023-12-03 11:15:35,450: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[2023-12-03 11:15:35,450: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] Incomplete yt initial data received. Retrying (1/3)...
[2023-12-03 11:15:35,578: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[2023-12-03 11:15:35,579: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] Incomplete yt initial data received. Retrying (2/3)...
[2023-12-03 11:15:35,724: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[2023-12-03 11:15:35,725: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] Incomplete yt initial data received. Retrying (3/3)...
[2023-12-03 11:15:35,843: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[2023-12-03 11:15:35,844: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] Incomplete yt initial data received; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U. Giving up after 3 retries
[2023-12-03 11:15:35,849: WARNING/ForkPoolWorker-16] ERROR: [youtube:tab] UCsBjURrPoezykLs9EqgamOA: Playlists that require authentication may not extract correctly without a successful webpage download. If you are not downloading private content, or your cookies are only for the first account and channel, pass "--extractor-args youtubetab:skip=authcheck" to skip this check
[2023-12-03 11:15:35,849: WARNING/ForkPoolWorker-16] https://www.youtube.com/channel/UCsBjURrPoezykLs9EqgamOA/videos: failed to get info from youtube with message ERROR: [youtube:tab] UCsBjURrPoezykLs9EqgamOA: Playlists that require authentication may not extract correctly without a successful webpage download. If you are not downloading private content, or your cookies are only for the first account and channel, pass "--extractor-args youtubetab:skip=authcheck" to skip this check
[2023-12-03 11:15:36,256: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[2023-12-03 11:15:36,258: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] Incomplete yt initial data received. Retrying (1/3)...
[2023-12-03 11:15:36,394: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[2023-12-03 11:15:36,395: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] Incomplete yt initial data received. Retrying (2/3)...
[2023-12-03 11:15:36,510: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[2023-12-03 11:15:36,510: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] Incomplete yt initial data received. Retrying (3/3)...
[2023-12-03 11:15:36,695: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[2023-12-03 11:15:36,696: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] Incomplete yt initial data received; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U. Giving up after 3 retries
[2023-12-03 11:15:36,701: WARNING/ForkPoolWorker-16] ERROR: [youtube:tab] UCsBjURrPoezykLs9EqgamOA: Playlists that require authentication may not extract correctly without a successful webpage download. If you are not downloading private content, or your cookies are only for the first account and channel, pass "--extractor-args youtubetab:skip=authcheck" to skip this check
[2023-12-03 11:15:36,701: WARNING/ForkPoolWorker-16] https://www.youtube.com/channel/UCsBjURrPoezykLs9EqgamOA/streams: failed to get info from youtube with message ERROR: [youtube:tab] UCsBjURrPoezykLs9EqgamOA: Playlists that require authentication may not extract correctly without a successful webpage download. If you are not downloading private content, or your cookies are only for the first account and channel, pass "--extractor-args youtubetab:skip=authcheck" to skip this check
[2023-12-03 11:15:37,092: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[2023-12-03 11:15:37,093: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] Incomplete yt initial data received. Retrying (1/3)...
[2023-12-03 11:15:37,216: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[2023-12-03 11:15:37,217: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] Incomplete yt initial data received. Retrying (2/3)...
[2023-12-03 11:15:37,337: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[2023-12-03 11:15:37,338: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] Incomplete yt initial data received. Retrying (3/3)...
[2023-12-03 11:15:37,537: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] unable to extract yt initial data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[2023-12-03 11:15:37,537: WARNING/ForkPoolWorker-16] WARNING: [youtube:tab] Incomplete yt initial data received; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U. Giving up after 3 retries
[2023-12-03 11:15:37,541: WARNING/ForkPoolWorker-16] ERROR: [youtube:tab] UCsBjURrPoezykLs9EqgamOA: Playlists that require authentication may not extract correctly without a successful webpage download. If you are not downloading private content, or your cookies are only for the first account and channel, pass "--extractor-args youtubetab:skip=authcheck" to skip this check
[2023-12-03 11:15:37,542: WARNING/ForkPoolWorker-16] https://www.youtube.com/channel/UCsBjURrPoezykLs9EqgamOA/shorts: failed to get info from youtube with message ERROR: [youtube:tab] UCsBjURrPoezykLs9EqgamOA: Playlists that require authentication may not extract correctly without a successful webpage download. If you are not downloading private content, or your cookies are only for the first account and channel, pass "--extractor-args youtubetab:skip=authcheck" to skip this check
[2023-12-03 11:15:37,576: WARNING/ForkPoolWorker-16] 56b049b9-ce3b-4f99-a189-3c6343458a9f success callback
[2023-12-03 11:15:37,581: INFO/ForkPoolWorker-16] Task extract_download[56b049b9-ce3b-4f99-a189-3c6343458a9f] succeeded in 2.630632034037262s: None
[2023-12-03 11:15:37,581: WARNING/ForkPoolWorker-16] 56b049b9-ce3b-4f99-a189-3c6343458a9f return callback

```

**Comment by bbilly1** [comment]
Thanks for looking into this. I'm pretty sure you are getting blocked by youtube, started collecting that here: https://docs.tubearchivist.com/faq/#am-i-getting-blockedthrottled

Works fine for me:
```
tubearchivist  | https://www.youtube.com/@Fireship
tubearchivist  | [2023-12-03 17:19:43,485: INFO/MainProcess] Task subscribe_to[e86dcc35-5db1-4e0b-a950-1be78462c851] received
tubearchivist  | [2023-12-03 17:19:43,486: WARNING/ForkPoolWorker-4] subscribe_to create callback
tubearchivist  | [2023-12-03 17:19:49,956: WARNING/ForkPoolWorker-4] UCsBjURrPoezykLs9EqgamOA: get metadata from es
tubearchivist  | [2023-12-03 17:19:49,960: WARNING/ForkPoolWorker-4] {"_index":"ta_channel","_id":"UCsBjURrPoezykLs9EqgamOA","found":false}
tubearchivist  | [2023-12-03 17:19:49,960: WARNING/ForkPoolWorker-4] UCsBjURrPoezykLs9EqgamOA: get metadata from youtube
tubearchivist  | [2023-12-03 17:19:50,584: WARNING/ForkPoolWorker-4] UCsBjURrPoezykLs9EqgamOA: download channel thumbnail
tubearchivist  | [2023-12-03 17:19:56,531: WARNING/ForkPoolWorker-4] UCsBjURrPoezykLs9EqgamOA: retry thumbnail download https://yt3.googleusercontent.com/nQxZwEfVWgF8zhdqc3gWTFmnDY21uPt6KMxw0m0-pRgW_lOt8jjAAfmFsC6ZRkMjZO-MeCxN6Q=s0
tubearchivist  | [2023-12-03 17:19:58,657: WARNING/ForkPoolWorker-4] e86dcc35-5db1-4e0b-a950-1be78462c851 success callback
tubearchivist  | [2023-12-03 17:19:58,659: INFO/ForkPoolWorker-4] Task subscribe_to[e86dcc35-5db1-4e0b-a950-1be78462c851] succeeded in 15.171181465999325s: None
tubearchivist  | [2023-12-03 17:19:58,659: WARNING/ForkPoolWorker-4] e86dcc35-5db1-4e0b-a950-1be78462c851 return callback
```

**Comment by gremi-jr** [comment]
Works also now for me!

**Comment by JasonLandbridge** [comment]
@bbilly1 , thanks! This was indeed the problem! I setup GluetunVPN and connected TubeArchivist to it to change the IP. To clear the cookie, I installed the [tubearchivist browser-extension](https://github.com/tubearchivist/browser-extension) in a browser I was not using and synced the YouTube logged out cookie from there :+1: 

Now everything is working again!

**Comment by derekslenk** [comment]
Using the testing branch, I still get the bool error on this channel: https://www.youtube.com/@slenk-jk8ty

**Comment by bbilly1** [comment]
looks like this needs additional handling in the `urlparser` module, same problem.

**Comment by gremi-jr** [comment]
When I use `https://www.youtube.com/@slenk-jk8ty` it will fail with `"Task failed: 'bool' object has no attribute 'get'"`. 

But when I use `https://www.youtube.com/@slenk-jk8ty/featured` it will work. Same with other Channels.

**Comment by derekslenk** [comment]
Is it expected that we paste in the featured page for channels?

**Comment by gremi-jr** [comment]
Dont think so. Like billy said, there is additional handling needed. I would assume that the program should do that for you.

**Comment by bbilly1** [comment]
Good news, that is now fixed upstream in yt-dlp directly. So I reverted our only partially working workaround. unstable build is out, with yt-dlp built from specific commit.

**Comment by MerlinScheurer** [comment]
I think i got the same or at least a similar problem...
With the latest unstable ( 5d2d6c1f8f3b ) i still get this error  ( 'bool' object is not subscriptable )

when i try to subscribe to all of this channels playlists ( only 2 of 3 work.   playlist: "uplift" gets added to the playlists, but with channel: "None" and when you click it, its "Not Found<br/> The requested resource was not found on this server." )

And it does not work when adding the "uplift" playlist to subscribes playlists.

The channel in question ( one of my channels ): https://www.youtube.com/@randomguy-jb5ts

( Not trying to block this, just trying to figure out if its related. )

**Comment by bbilly1** [comment]
works fine for me:
```
tubearchivist  | https://www.youtube.com/playlist?list=PLny6D86FY8sFKD9NyWOMtsd4VVoAhOLBJ
tubearchivist  | [2023-12-17 18:19:47,081: INFO/MainProcess] Task subscribe_to[aa1e6290-88f3-4d17-bceb-ebb13783a1c9] received
tubearchivist  | [2023-12-17 18:19:47,082: WARNING/ForkPoolWorker-4] subscribe_to create callback
tubearchivist  | [2023-12-17 18:19:47,112: WARNING/ForkPoolWorker-4] PLny6D86FY8sFKD9NyWOMtsd4VVoAhOLBJ: get metadata from es
tubearchivist  | [2023-12-17 18:19:47,115: WARNING/ForkPoolWorker-4] {"_index":"ta_playlist","_id":"PLny6D86FY8sFKD9NyWOMtsd4VVoAhOLBJ","found":false}
tubearchivist  | [2023-12-17 18:19:47,115: WARNING/ForkPoolWorker-4] PLny6D86FY8sFKD9NyWOMtsd4VVoAhOLBJ: get metadata from youtube
tubearchivist  | [2023-12-17 18:19:48,935: WARNING/ForkPoolWorker-4] UCf3j6ENJGKipNw9B8etIdig: get metadata from es
tubearchivist  | [2023-12-17 18:19:48,939: WARNING/ForkPoolWorker-4] {"_index":"ta_channel","_id":"UCf3j6ENJGKipNw9B8etIdig","found":false}
tubearchivist  | [2023-12-17 18:19:48,939: WARNING/ForkPoolWorker-4] UCf3j6ENJGKipNw9B8etIdig: get metadata from youtube
tubearchivist  | [2023-12-17 18:19:49,296: WARNING/ForkPoolWorker-4] UCf3j6ENJGKipNw9B8etIdig: download channel thumbnail
tubearchivist  | [2023-12-17 18:19:49,914: WARNING/ForkPoolWorker-4] UCf3j6ENJGKipNw9B8etIdig: failed to extract thumbnail, use fallback
tubearchivist  | [2023-12-17 18:19:49,920: WARNING/ForkPoolWorker-4] UCf3j6ENJGKipNw9B8etIdig: failed to extract thumbnail, use fallback
tubearchivist  | [2023-12-17 18:19:50,240: WARNING/ForkPoolWorker-4] aa1e6290-88f3-4d17-bceb-ebb13783a1c9 success callback
tubearchivist  | [2023-12-17 18:19:50,241: INFO/ForkPoolWorker-4] Task subscribe_to[aa1e6290-88f3-4d17-bceb-ebb13783a1c9] succeeded in 3.1580140739999933s: None
tubearchivist  | [2023-12-17 18:19:50,242: WARNING/ForkPoolWorker-4] aa1e6290-88f3-4d17-bceb-ebb13783a1c9 return callback
```

Are you sure you pulled the newest unstable? From just 6h ago. Or what do you mean with `5d2d6c1f8f3b`? Commit should be https://github.com/tubearchivist/tubearchivist/commit/ddc0b7a481a1906ca84de430235c87ada5c1e253 or docker image digest `ce614eb8a764`.

**Comment by MerlinScheurer** [comment]
It should be the latest unstable. ( local time. ~1h ago)  the hash is what watchtower reported as the hash. ( not sure where its gets that from )
```
watchtowerOnce  | time="2023-12-17T11:42:05+01:00" level=info msg="Found new bbilly1/tubearchivist:unstable image (5d2d6c1f8f3b)"
watchtowerOnce  | time="2023-12-17T11:42:14+01:00" level=info msg="Stopping /tubearchivist (cf61b3040a2a) with SIGTERM"
watchtowerOnce  | time="2023-12-17T11:42:24+01:00" level=info msg="Creating /tubearchivist"
```

Is the playlist in your subscribed playlists and is the channel set?

**Comment by gremi-jr** [comment]
Works also for me with all the playlists from your channel, from mine and the original provided on in this issue above. 

![grafik](https://github.com/tubearchivist/tubearchivist/assets/50887574/a7fc876b-2bd7-4200-9686-13b9a995649a)
![grafik](https://github.com/tubearchivist/tubearchivist/assets/50887574/f3b86fb8-975f-4205-bd41-b9b8e7e0c1d2)

**Comment by MerlinScheurer** [comment]
@gremi-jr interesting...  maybe something is broken in my config 🤔

I will have a look into my elasticsearch, maybe there is a broken channel with the name "None" somewhere.

So, lets say my problem is unrelated and the original problem is fixed 😉

**Comment by gremi-jr** [comment]
@MerlinScheurer it is releated to this problem I think. Because I had the same result from what you described with the current stable Build. To fix this I just wiped everything and started from scratch.

**Comment by MerlinScheurer** [comment]
@gremi-jr you are right it was the same issue, but updating to the latest unstable did not solve it.  ( it should not appear again, though )

I just checked in my elasticsearch with kibana and it seems that somehow this playlist got indexed with playlist_channel = null
 and playlist_channel_id = null .
This caused it to fail and the channel being defaulted to "None", which put the playlist in a weird state of being there but not assigned to any channel.
Which is not solvable by performing a re-index on that channel. 😕 

@bbilly1 the issue i described happened on the latest unstable ( checked that 4 times now to be sure 😁 ), BUT its a side effect of being affected by this issue in the first place.

Solution to my issue:
I had to send a manual playlist delete request to the tubearchivist api with the playlist id.
( https://<address:port>/api/playlist/<playlistID>/ )
That solved it and the playlist now behaves as expected 😄 


tl;dr:
Update seems fine.
If you are affected by a playlist being of channel "None", use the api to delete it and then add it again.

**Comment by bbilly1** [comment]
Just manually recreated the missing channel on playlist scenario. Refreshing the channel is indeed not solving the problem. Refreshing the playlist though is. Unfortunatly the playlist page will become inaccessible without a channel, but you can still refresh it over the API:

```
POST /api/refresh/
{"playlist": ["<playlist-id>"]}
```

so you don't have to delete it. Also periodic refresh will fix that eventually as well...

#620 introduces custom playlist will also implement playlists without channels, making that more robust. But I don't want to hold of the release, as this ships the fix avoiding that problem in the first place.

**Comment by bbilly1** [comment]
merged in v0.4.5, shout out to the amazing yt-dlp team for creating a fix upstream.

**Comment by TomTomTomGit** [comment]
I almost don't dare to add to this old threat.. but I've just installed the Archivist docker image(s) on my Unraid Server and got the very same issue :-(

**Comment by Kuroo2021** [comment]
Same as above some channels I sub to but when trying https://www.youtube.com/@AsuraChadRecap it doesn't work.
